tor/core/inbox.py

```diff
@@ -58,6 +58,9 @@ def forward_to_slack(item: InboxableMixin, cfg: Config) -> None:
 
 
 @beeline.traced(name="process_reply")
+def extract_sub_from_url(url: str) -> str:
+    # returns the sub name from the given url without "r/" at the start.
```

Comment:
If you document what a function does you should do it as a so-called "doc-string". It's basically just a string as the first item in a function, which has a special meaning.
Then your IDE will also show this string when you hover over the function name.

```suggestion
    """returns the sub name from the given url without "r/" at the start."""
```

Reply:
Oh this is nice actually!